api/src/main/java/jakarta/data/repository/Query.java

```diff
@@ -114,8 +114,8 @@
      *
      * <p>The default value of empty string indicates that no counting query
      * is provided. A counting query is unnecessary when pagination is
-     * performed with slices instead of pages and when pagination is
-     * not used at all.</p>
+     * performed with {@link jakarta.data.page.PageRequest#withTotal()} or
+     * when pagination is not used at all.</p>
```

Comment:
```suggestion
     * performed with {@link jakarta.data.page.PageRequest#withoutTotal()} or
     * when pagination is not used at all.</p>
```

Reply:
I could _swear_ that this is what I typed!